packages/vite/src/hbs.ts

```diff
@@ -14,9 +14,15 @@ export function hbs(): Plugin {
   return {
     name: 'rollup-hbs-plugin',
     enforce: 'pre',
-    async resolveId(source: string, importer: string | undefined) {
+    async resolveId(source: string, importer: string | undefined, options) {
+      if (options.custom?.onlyResolver) {
```

Comment:
hbs resolveId and template-tag resolveId are invoking each other in vite build.
checking enableCustomResolver: false here will cause optmize deps issues.
@ef4 maybe you can have a look here too :)